files/wordpress/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3.1'
+
+services:
+
+  wordpress:
+    image: wordpress:4.9.1
+    restart: always
+    depends_on:
+      - mysql
+    ports:
+      - 8080:80
+    environment:
+      WORDPRESS_DB_PASSWORD: ${MYSQL_PASSWORD:-testing123}
+      WORDPRESS_DB_HOST: ${WORDPRESS_DB_HOST:-mysql}
+      WORDPRESS_DB_NAME: ${MYSQL_DATABASE:-openmrs}
+      WORDPRESS_DB_USER: ${WORDPRESS_DB_USER:-openmrs}
+      WORDPRESS_AUTH_KEY: ${WORDPRESS_AUTH_KEY:-authkeysalt}
+      WORDPRESS_SECURE_AUTH_KEY: ${WORDPRESS_SECURE_AUTH_KEY:-secureauthkeysalt}
+      WORDPRESS_LOGGED_IN_KEY: ${WORDPRESS_LOGGED_IN_KEY:-loggedinkeysalt}
+      WORDPRESS_NONCE_KEY: ${WORDPRESS_NONCE_KEY:-noncekeysalt}
+      WORDPRESS_AUTH_SALT: ${WORDPRESS_AUTH_SALT:-authsalt}
+      WORDPRESS_SECURE_AUTH_SALT: ${WORDPRESS_SECURE_AUTH_SALT:-secureauthsalt}
+      WORDPRESS_LOGGED_IN_SALT: ${WORDPRESS_LOGGED_IN_SALT:-loggedinsalt}
+      WORDPRESS_NONCE_SALT: ${WORDPRESS_NONCE_SALT:-noncesalt}
+    volumes:
+      - wordpress-data:/var/www/html
+      - ./dbdump:/docker-entrypoint-initdb.d
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost/"]
+
+  mysql:
+    image: mysql:5.6
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-testingroot123}
+      MYSQL_PASSWORD: ${MYSQL_PASSWORD:-testing123}
+      MYSQL_USER: ${MYSQL_USER:-openmrs}
```

Comment:
Can you use another name, like 'wordpress' for the database?

Reply:
changed to wordpress